tailscale/client.go

```diff
@@ -359,25 +360,49 @@ func (c *Client) DeviceSubnetRoutes(ctx context.Context, deviceID string) (*Devi
 	return &resp, nil
 }
 
+// MaybeEmptyTime wraps a time and allows for unmarshalling timestamps that represent an empty time as an empty string (e.g "")
+// this is used by the tailscale API when it returns services that have no created date, such as it's hello service.
+type MaybeEmptyTime struct {
```

Comment:
I think I'd prefer if this type was just called `Time`, so that when imported and used it could be `tailscale.Time`, it sounds a little better

Reply:
Sounds good! Let me make this change now